tests/trove/derive/test_osfmap_json.py

```diff
@@ -1,548 +1,559 @@
-                    "identifier": [
-                        {"@value": "https://ror.example/05d5mza29"}
-                    ],
-                    "name": [
-                        {"@value": "Center For Open Science"}
-                    ]
-                }
-            ],
-            "archivedAt": [
-                {"@id": "https://archive.example/details/osf-registrations-2c4st-v1"}
-            ],
-            "osf:contains": [
-                {
-                    "@id": "https://osf.example/2ph9b",
-                    "resourceType": [
-                        {"@id": "File"}
-                    ],
-                    "dateCreated": [
-                        {"@value": "2021-10-18"}
-                    ],
-                    "identifier": [
-                        {"@value": "https://osf.example/2ph9b"}
-                    ],
-                    "dateModified": [
-                        {"@value": "2021-10-18"}
-                    ],
-                    "fileName": [
-                        {"@value": "test_file.txt"}
-                    ],
-                    "filePath": [
-                        {"@value": "/Archive of OSF Storage/test_file.txt"}
-                    ],
-                    "isContainedBy": [
-                        {"@id": "https://osf.example/2c4st"}
-                    ]
-                }
-            ],
-            "hostingInstitution": [
-                {
-                    "@id": "https://cos.example/",
-                    "resourceType": [
-                        {"@id": "Agent"},
-                        {"@id": "Organization"}
-                    ],
-                    "identifier": [
-                        {"@value": "https://cos.example/"},
-                        {"@value": "https://ror.example/05d5mza29"}
-                    ],
-                    "sameAs": [
-                        {
-                            "@id": "https://ror.example/05d5mza29",
-                            "resourceType": [
-                                {"@id": "Agent"},
-                                {"@id": "Organization"}
-                            ],
-                            "identifier": [
-                                {"@value": "https://ror.example/05d5mza29"}
-                            ],
-                            "name": [
-                                {"@value": "Center For Open Science"}
-                            ]
-                        }
-                    ],
-                    "name": [
-                        {"@value": "Center for Open Science"}
-                    ]
-                }
-            ],
-            "keyword": [
-                {"@value": "Demo"},
-                {"@value": "IA"},
-                {"@value": "IMLS"},
-                {"@value": "OSF"}
-            ]
-        },
-    }
+
+class TestIndexcardJsonDeriver(BaseIndexcardDeriverTest):
+    deriver_class = IndexcardJsonDeriver
+    expected_outputs = expected_outputs
```

Comment:
alas, if these two derivers were supposed to have the same outputs we wouldn't need both of them -- add separate `expected_outputs` for the "mini" to check it is correctly pruned (which it's not, thanks to that `return True if ... else True`)